Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:latest
+MAINTAINER nelluri@deis.com
+RUN apt-get update && apt-get -y install make python lsb-release unzip wget vim git curl jq
+RUN wget https://storage.googleapis.com/kubernetes-release/release/v1.0.3/bin/linux/amd64/kubectl
+RUN chmod +x kubectl
+RUN mv kubectl /usr/local/bin/
+WORKDIR /home
+RUN wget --no-check-certificate https://dl.bintray.com/deis/helm/helm-0.5.0+1689ee4-linux-amd64.zip
```

Comment:
My guess is you ran an apt-get install prior that didn't do "the right thing" try busting your Docker cache and trying again. I had good luck with the removal of the insecure flags on curl/wget.